lib/app/features/feed/create_article/views/pages/create_article_modal/components/create_article_toolbar.dart

```diff
@@ -0,0 +1,90 @@
+// SPDX-License-Identifier: ice License 1.0
+
+import 'package:flutter/material.dart';
+import 'package:flutter_hooks/flutter_hooks.dart';
+import 'package:flutter_quill/flutter_quill.dart';
+import 'package:ion/app/features/feed/views/components/actions_toolbar/actions_toolbar.dart';
+import 'package:ion/app/features/feed/views/components/text_editor/components/toolbar_buttons/toolbar_bold_button/toolbar_bold_button.dart';
```

Comment:
Let's create a barrel file for all these imports (for example, like it is in `feature/gallery/views/components/camera/camera.dart`), and could you please put the files related to `toolbar_buttons` into that folder instead of creating single folders for each single file?